CHANGELOG.rst

```diff
@@ -61,30 +60,13 @@ TBR
 
                 def parse(self, response: scrapy.http.Response, page: MyPage):
                     ...
-        
-      To avoid this issue, you can:
-
-        * avoid defining the ``parse()`` method and instead use any other name.
-
-            * This is the **recommended** approach since it's easier and avoids
-              any :class:`scrapy.http.Request` with ``callback`` set to ``None``
-              which could come from anywhere (e.g. introducing new 3rd-party
-              middlewares or pipelines).
-
-        * ensure that all your :class:`scrapy.http.Request` has a ``callback``
-          **not** set to ``None``.
```

Comment:
Won't users get warning from built-in components if they do it?

Reply:
Yeah, they'd still get warnings from things like RobotsMiddlewares